fiftyone/operators/delegated.py

```diff
@@ -80,7 +80,7 @@ def set_progress(self, doc_id, progress):
         """
         return self._repo.update_progress(_id=doc_id, progress=progress)
 
-    def set_running(self, doc_id, progress=None, run_link=None):
+    def set_running(self, doc_id, progress=None, run_link=None, required_state: ExecutionRunState = None):
```

Comment:
this line appears to be too long. did you run the precommit hooks which include black?

Reply:
Had it set to 99, whoops!

Reply:
yeah we use default of 79 in this repo. In others we have chosen to go with 99